alonerpg/views.py

```diff
@@ -8,19 +8,30 @@ class HomePageView(TemplateView):
 
     def get(self, request, *args, **kwargs):
         context = self.get_context_data(**kwargs)
-        if "command" in request.GET:
-            command = request.GET["command"]
+        rolled_table = ""
+
+        if "rtn" in request.GET:
+            command = "rtn " + request.GET["rtn"]
```

Comment:
If you decide to go with `f-strings`, we should refactor this as well for code consistency